build/generic/web/viewer.html

```diff
@@ -51,22 +53,22 @@
 
   </head>
 
-  <body tabindex="1" class="loadingInProgress">
+  <body tabindex="1" class="loadingInProgress" id="lightTheme">
     <div id="outerContainer">
 
-      <div id="sidebarContainer">
+      <div id="sidebarContainer" class="hiddenWhenSidebarUnavailable">
         <div id="toolbarSidebar">
-          <div class="splitToolbarButton toggled">
-            <button id="viewThumbnail" class="toolbarButton group toggled" title="Show Thumbnails" tabindex="2" data-l10n-id="thumbs">
-               <span data-l10n-id="thumbs_label">Thumbnails</span>
-            </button>
-            <button id="viewOutline" class="toolbarButton group" title="Show Document Outline" tabindex="3" data-l10n-id="outline">
-               <span data-l10n-id="outline_label">Document Outline</span>
-            </button>
-            <button id="viewAttachments" class="toolbarButton group hidden" title="Show Attachments" tabindex="4" data-l10n-id="attachments">
-               <span data-l10n-id="attachments_label">Attachments</span>
-            </button>
-          </div>
+         <button id="viewThumbnail" class="toolbarButton toggled" title="Show Thumbnails" tabindex="2" data-l10n-id="thumbs">
+<?= $thumbnailsIcon ?> 
+            <span data-l10n-id="thumbs_label">Thumbnails</span>
+         </button>
+         <button id="viewOutline" class="toolbarButton" title="Show Document Outline" tabindex="3" data-l10n-id="outline">
+<?= $listIcon ?> 
+            <span data-l10n-id="outline_label">Document Outline</span>
+         </button>
+         <button id="viewAttachments" class="toolbarButton group hidden" title="Show Attachments" tabindex="4" data-l10n-id="attachments">
```

Comment:
It doesn't really matter since it's hidden anyway.

Also, you can skip the commits that say "commit build files." They contain changes that have already been reviewed in other pull requests.